Bottom: f5bea57fbf958bf8040d2b2caf4fca773b03a305
Top:    7655233f2f581f084c661197bc4441ce346679dd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 10:38:29 +0000

Suggested fix to Select(NetByName,...)


---

diff --git a/src/select.c b/src/select.c
index bd4f373..71277b5 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1018,7 +1018,6 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       InitConnectionLookup ();
       ResetFoundPinsViasAndPads (false);
       ResetFoundLinesAndPolygons (false);
-      SaveUndoSerialNumber ();
 
       MENU_LOOP (&PCB->NetlistLib);
       {
@@ -1036,8 +1035,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
 	  }
       }
       END_LOOP;
-      RestoreUndoSerialNumber ();
-      SelectConnection (Flag);
+      changed = SelectConnection (Flag);
       ResetFoundPinsViasAndPads (false);
       ResetFoundLinesAndPolygons (false);
       FreeConnectionLookupMemory ();
